source/common/dynamo/dynamo_filter.h

```diff
@@ -27,7 +27,8 @@ class DynamoFilter : public Http::StreamFilter {
   }
 
   // Http::StreamFilterBase
-  void onDestroy() override {}
+  void setBufferLimit(uint32_t limit) override { buffer_limit_ = limit; }
+  void onDestroy() override { stream_reset_ = true; }
```

Comment:
Can `decodeData` ever be called after `onDestroy`? If not, then `stream_reset_` isn't doing anything.